azure-pipelines/main-pipeline.yml

```diff
@@ -130,13 +133,13 @@ stages:
 
           - task: SonarCloudPrepare@1
             inputs:
-              SonarCloud: 'SonarCloud'
-              organization: '$(sonarOrg)'
-              scannerMode: 'CLI'
-              configMode: 'manual'
-              cliProjectKey: '$(sonarKeyPrefix)base'
-              cliProjectName: 'base'
-              cliSources: 'base'
+              SonarCloud: "SonarCloud"
+              organization: "$(sonarOrg)"
+              scannerMode: "CLI"
+              configMode: "manual"
+              cliProjectKey: "$(sonarKeyPrefix)base"
+              cliProjectName: "base"
+              cliSources: "base"
```

Comment:
not really a useful diff, and will just go back to single quotes next time someone uses the task assistant to edit this step. definitely non-blocking issue.

Reply:
VScode did this when I checked the formatting, not intentional but I didn't think it affected functionality in this scenario.